BitwardenShared/Core/Auth/Services/AuthService.swift

```diff
@@ -681,6 +724,12 @@ class DefaultAuthService: AuthService { // swiftlint:disable:this type_body_leng
         // Get the identity token to log in to Bitwarden.
         let response = try await getIdentityTokenResponse(email: email, request: twoFactorRequest)
 
+        // If it's assigned then we need to update the required reset password and remove the cache.
+        if forcePasswordResetReason != nil {
+            try await stateService.setForcePasswordResetReason(.weakMasterPasswordOnLogin)
+            forcePasswordResetReason = nil
```

Comment:
🤔 If the user chooses not to login with the current account, when is `forcePasswordResetReason` cleared?